pkg/core/proxy/mockmanager.go

```diff
@@ -2,33 +2,33 @@ package proxy
 
 import (
 	"fmt"
+	"sort"
+	"strconv"
+	"strings"
 	"sync"
 
 	"go.keploy.io/server/v2/pkg/models"
 	"go.uber.org/zap"
 )
 
 type MockManager struct {
-	filtered                *TreeDb
-	unfiltered              *TreeDb
-	logger                  *zap.Logger
-	utilizedFilteredMocks   sync.Map
-	utilizedUnFilteredMocks sync.Map
+	filtered      *TreeDb
+	unfiltered    *TreeDb
+	logger        *zap.Logger
+	consumedMocks sync.Map
 }
 
 func NewMockManager(filtered, unfiltered *TreeDb, logger *zap.Logger) *MockManager {
 	return &MockManager{
-		filtered:                filtered,
-		unfiltered:              unfiltered,
-		logger:                  logger,
-		utilizedFilteredMocks:   sync.Map{},
-		utilizedUnFilteredMocks: sync.Map{},
+		filtered:      filtered,
+		unfiltered:    unfiltered,
+		logger:        logger,
+		consumedMocks: sync.Map{},
 	}
 }
 
 func (m *MockManager) SetFilteredMocks(mocks []*models.Mock) {
 	m.filtered.deleteAll()
-	m.utilizedFilteredMocks = sync.Map{}
```

Comment:
I think it should clear the consumed mocks to print the mocks for a test-case run. Otherwise, mocks of test-1 will also be logged as consumed mocks for test-2

Reply:
clearing it in getconusmedMocks only, If we clear it in setfiltered mocks we will miss the mocks used up just after starting app and before running 1st test case

Reply:
Got it. That is correct.